python/tank/util/local_file_storage.py

```diff
@@ -74,8 +74,24 @@ def get_global_root(cls, path_type, generation=CORE_V18):
         """
         if generation == cls.CORE_V18:
 
+            # If the environment variable is available and set to an actual value.
+            shotgun_home_override = os.environ.get("SHOTGUN_HOME")
+            if shotgun_home_override:
+                # Make sure the path is an absolute path.
+                shotgun_home_override = os.path.abspath(shotgun_home_override)
```

Comment:
I think it would be cool to be able to set `SHOTGUN_HOME=~/data/shotgun` or `SHOTGUN_HOME=/prod/$USER/shotgun`. can we include the resolution of env vars and ~ here as well?